assets/dev/js/editor/elements/views/widget.js

```diff
@@ -228,7 +228,7 @@ WidgetView = BaseElementView.extend( {
 		// TODO: Find a better way to detect if all the images have been loaded
 		self.$el.imagesLoaded().always( function() {
 			setTimeout( function() {
-				if ( 1 > self.$el.children( '.elementor-widget-container' ).outerHeight() ) {
```

Comment:
What was changed here?

Reply:
Please look at ED.

Reply:
jQuery 1.12: `self.$el.children( '.elementor-widget-container' ).outerHeight() === null` 
jQuery 3.5: `self.$el.children( '.elementor-widget-container' ).outerHeight() === undefined`

Reply:
So a better check is just `if ( self.$el.children( '.elementor-widget-container' ).outerHeight()`

Reply:
So a better check is just `if ( self.$el.children( '.elementor-widget-container' ).outerHeight()`

Reply:
@kobizz, its not because: it will never work...
![image](https://user-images.githubusercontent.com/10234691/104836972-446baa00-58ba-11eb-98ca-463a45c03e42.png)
i think it was uses `1 >` for the situations like bigger then `1` or bigger then `null`.

Reply:
It will work because both `null` and `undefined` won't pass the condition